app/views/providers/check_provider_answers/index.html.erb

```diff
@@ -2,10 +2,10 @@
   content_for(:navigation) do
     link_to(
       t('generic.back'),
-      reset_providers_legal_aid_application_check_provider_answers_path,
+      @back_step_url,
```

Comment:
If you remove the `reset...`, the application is stuck in "check_answers" state.
When on "Check answers" page, try clicking "back" twice, you should go back 2 steps (to `other_asserts` or `savings_and_investments`).
But with this change you go back to the "Check answers" page.

Reply:
yep